README.md

```diff
@@ -197,6 +197,7 @@ The following actions will trigger this event:
 | ---------------------- | ------------------------- | ------------------------------------------------------------ | ---------------------------------------------------------------------- |
 | openBindingEvent       | `string`                  | The MouseEvent type to display the editMode                  | `click`                                                              |
 | closeBindingEvent      | `string`                  | The MouseEvent type to display the viewMode                  | `click`                                                                 |
+| enabled                | `boolean`                 | Allows or forbids edit mode (doesn't swith to it)            | `true`                                                                 |
```

Comment:
sorry, one last change as there is a typo: from `swith` to `switch`

Reply:
👍 right